### DISCUSSION TITLE
Sending image binary buffer example (tx in advance)

### DISCUSSION THREAD

**Discussion by alsak0de** [WhiskeySockets/Baileys#324] [General Questions]
Hello, could anyone provide an example of a msg json structure where the message to be sent is expressed as a binary buffer and not a url?. Ideally, I'd like to see the data in the request and not as the result of a file read to variable. Would the buffer be expressed as integer array?

msg  = {
"image": [0,0,0,12,12,34,345, ...., 0]
}

or.. does it have to be converted to a b64 string?

**Comment by xirf** [comment]
You can just past the buffer without specifying the URL
```
await socket.sendMessage(jid, {
            image: path,
            caption
});
```
